docs/mobile/quick-start.mdx

```diff
@@ -106,28 +105,26 @@ The constructors of the `App.xaml.cs` and `MainPage.xaml.cs` files in the UWP sp
 
 ## Error Handling
 
-In each of the platform specific setups, there is a call to the `HandleExceptions` method. This method tells Geocortex Mobile to handle exceptions and display an Error. If you need to have control over global error handling, you can implement your own global exception handling for the Xamarin Application. [This Stack Overflow post](https://stackoverflow.com/questions/39503390/global-exception-handling-in-xamarin-cross-platform) is a good starting point for implementating global error handling.
+In each of the platform specific setups, there is a call to the `HandleExceptions` method. This method tells Geocortex Mobile to handle exceptions and display an Error. If you need to have control over global error handling, you can remove the call to `HandleExceptions` and implement your own global exception handling for the Xamarin Application. [This Stack Overflow post](https://stackoverflow.com/questions/39503390/global-exception-handling-in-xamarin-cross-platform) is a good starting point for implementating global error handling.
 
 ## Editing the Layout and App Config
 
-Many changes to Geocortex Mobile will require editing the layout and app config for your custom application. The app config and layout files are contained in the shared platform agnostic project. Each of the platform specific projects links to these files as an asset or a reference, so editing the files in the platform agnostic project will change them for all platforms. Check out [layout](concepts-layout.mdx) and [app config](concepts-app-config.mdx) to learn more about customizing Geocortex Mobile with these files.
+Many changes to Geocortex Mobile will require editing the layout and app config for your custom application. The app config and layout files are contained in the shared platform agnostic project. Each platform specific project links to these files as an asset or a reference, so editing the files in the platform agnostic project will change them for all platforms. Check out [layout](concepts-layout.mdx) and [app config](concepts-app-config.mdx) to learn more about configuring Geocortex Mobile with these files.
 
 ## Referencing Third Party Libraries
 
-The [Geocortex Mobile quick start](../mobile/quick-start.mdx) is simply a [.NET Xamarin Forms project](https://docs.microsoft.com/en-us/xamarin/get-started/quickstarts/deepdive?pivots=windows) so you can add dependencies to it like any other .NET project.
+The [Geocortex Mobile quickstart](../mobile/quick-start.mdx) is simply a [.NET Xamarin Forms project](https://docs.microsoft.com/en-us/xamarin/get-started/quickstarts/deepdive?pivots=windows) so you can add dependencies to it like any other .NET project. However, the Geocortex Mobile SDK relies on exact versions of [ArcGIS Runtime](https://developers.arcgis.com/arcgis-runtime/) and [Xamarin Forms](https://dotnet.microsoft.com/apps/xamarin/xamarin-forms) NuGet packages.
```

Comment:
I would move the "However...." section into a :::caution tag
https://v2.docusaurus.io/docs/markdown-features#calloutsadmonitions

Reply:
Moved the text into a :::caution tag.